module/signature/single.go

```diff
@@ -0,0 +1,25 @@
+// +build relic
+
+package signature
+
+import (
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/crypto/hash"
+	"github.com/onflow/flow-go/module"
+)
+
+type SingleSigner struct {
```

Comment:
minimal godoc, does this implement `MsgSigner` interface?

Reply:
Good point, updated